ui/analyse/src/crazy/crazyView.ts

```diff
@@ -11,16 +13,18 @@ type Position = 'top' | 'bottom';
 
 export default function (ctrl: AnalyseCtrl, color: Color, position: Position) {
   if (!ctrl.node.crazy || ctrl.data.game.variant.key !== 'crazyhouse') return;
+  const pos = setupPosition('crazyhouse', parseFen(ctrl.node.fen).unwrap()).unwrap();
   const pocket = ctrl.node.crazy.pockets[color === 'white' ? 0 : 1];
   const dropped = ctrl.justDropped;
   const captured = ctrl.justCaptured;
   if (captured) captured.role = captured.promoted ? 'pawn' : captured.role;
   const activeColor = color === ctrl.turnColor();
-  const usable = !ctrl.embed && activeColor;
+  const usable = !ctrl.embed && activeColor && !pos.isEnd();
```

Comment:
This alone seems good enough. I don't think we need or even want to hide the pockets when the game is over.

Reply:
I can a send screenshot but the current behaviour in this PR is to still show the pocket but grey out the pieces after the game has ended. Should I still remove this?

Reply:
Well, Thibault already adjusted it now anyway but the style you used is exactly the same as pockets that don't have any pieces and you also hid the count? I guess "hide the pockets" was a bad way to phrase it but it would have effectively hidden the pieces in the pockets.